src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/UserManagementSteps.java

```diff
@@ -161,4 +161,33 @@ public void retireMessage() throws Exception {
         accountsPage.containsText("Retire avec success");
     }
 
+    @And("Click the pencil icon in the ‘Action’ column next to the preffred user name")
+    public void clickPencilIcon() throws Exception {
+        manageAccountsPage.goToAccountsPage();
+    }
+
+    @And("Select the modifier icon in the top right corner of the ‘details du compte d’utilisateur’")
+    public void selectModifierIcon() throws Exception {
+        accountsPage.clickEditButton();
+    }
+
+    @And("select the appropriate privileges under ‘capacites’ checkboxes {string} {string}")
+    public void selectAppropriatePrivillages(String userName, String privillage) throws Exception {
+        accountsPage.enterUserName(userName);
+        accountsPage.selectPrevillageLevel(privillage);
+        Thread.sleep(5000);
```

Comment:
These thread sleeps are not good to use , we better use selenium time waits utils

Reply:
Thats true @sherrif10 .
The selenium wait polls the DOM based on a given condition , while the Thread.sleep ,sleeps the script without condition .
We shall refactor these out

Reply:
thanks guys for the reviews